src/main/java/de/lukweb/justmail/smtp/SmtpResponse.java

```diff
@@ -20,7 +20,7 @@
     ARGUMENT_ERROR(501, "Synatx error in parameters or arguments"),
     COMMAND_NOT_IMPLEMENTED(502, "SmtpCommand not implemented"),
     BAD_SEQUENCE(503, "Bad sequence of commands"),
-    ARGUMENT_NOT_IMPLEMENTED(504, "SmtpCommand parameter not implemented"),
```

Comment:
But these are the enhanced status codes, not the one that were originally defined in the SMTP protocol and the goal of the SMTP part of this project is to fully implement the SMTP protocol.